lib/src/server/handler.dart

```diff
@@ -146,7 +148,6 @@ class ServerHandler extends ServiceCall {
   }
 
   void _startStreamingRequest() {
-    _incomingSubscription.pause();
```

Comment:
Is this change related to the interceptors?

Reply:
It's related to async interceptor (and perhaps more as #108 reports the same error I have before the last commit ffe489b)